modules/planning/tasks/deciders/open_space_fallback_decider.cc

```diff
@@ -54,24 +57,60 @@ Status OpenSpaceFallbackDecider::Process(Frame* frame) {
     auto ptr_fallback_trajectory_pair =
         frame_->mutable_open_space_info()->mutable_fallback_trajectory();
 
-    double stop_distance =
-        std::max(0.0, obstacle_to_vehicle_distance -
-                          config_.open_space_fallback_decider_config()
-                              .open_space_fall_back_stop_safety_gap());
-    if (stop_distance > 0.0) {
-      // the accelerate = v0^2 / (2*s), where s is slowing down distance
-      // TODO(Runxin): a better fallback trajectory
+    double relative_collision_point_x = ptr_fallback_trajectory_pair->
+        first[first_collision_idx].path_point().x() -
+        ptr_fallback_trajectory_pair->first[0].path_point().x();
+    double relative_collision_point_y = ptr_fallback_trajectory_pair->
+        first[first_collision_idx].path_point().y() -
+        ptr_fallback_trajectory_pair->first[0].path_point().y();
```

Comment:
Save `ptr_fallback_trajectory_pair->first[first_collision_idx].path_point()` and `ptr_fallback_trajectory_pair->first[0].path_point()` to var and reuse.

Reply:
> Save `ptr_fallback_trajectory_pair->first[first_collision_idx].path_point()` and `ptr_fallback_trajectory_pair->first[0].path_point()` to var and reuse.

fixed